Lesson 3/src/Login.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+class Login extends React.Component{
+    render(){
+        const [mail,pwd,button] = this.props.items;
+        // const titleLogo = this.props.titleLogo;
+        const titleLogo = this.props.children;
```

Comment:
```suggestion
        const {children: titleLogo} = this.props;
```
так мы сможем сразу в одном месте вытаскивать props'ы (в дальнейшем)